TestName,Result,Message
test_Tainted_A_cwe078_2.TestCWE078_2.test_correctness,failure,"Expected 'system' to be called once. Called 2 times.
Calls: [call('conda update -n jupyter notebook librosa sox\n                 conda install -y librosa=0.7.0 sox=1.3.1\n                '),
 call('conda install -y github')]."
test_Tainted_A_cwe078_2.TestCWE078_2.test_vulnerability,failure,200 != 500
